framework/src/media/audio/audio_manager.c

```diff
@@ -28,71 +28,74 @@
 #include <tinyalsa/tinyalsa.h>
 
 #include "audio_manager.h"
+#include "resample/samplerate.h"
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define MAX_AUDIO_DEVICE_NUM			3
-#define AUDIO_STREAM_CHANNEL_MONO			1
-#define AUDIO_STREAM_CHANNEL_STEREO			2
+#define MAX_AUDIO_DEVICE_NUM 3
+#define AUDIO_STREAM_CHANNEL_MONO 1
+#define AUDIO_STREAM_CHANNEL_STEREO 2
 // ToDO: Expand with policies, Large data requires a large period.
-#define AUDIO_STREAM_VOIP_PERIOD_SIZE			512
-#define AUDIO_STREAM_VOIP_PERIOD_COUNT			8
-#define AUDIO_STREAM_VOIP_SAMPLE_RATE			16000
+#define AUDIO_STREAM_VOIP_PERIOD_SIZE 2048
+#define AUDIO_STREAM_VOIP_PERIOD_COUNT 8
+#define AUDIO_STREAM_VOIP_SAMPLE_RATE 16000
 
-#define AUDIO_STREAM_RETRY_COUNT			2
+#define AUDIO_STREAM_RETRY_COUNT 2
 
-#define AUDIO_DEVICE_UNDEF			0x0000
-#define AUDIO_DEVICE_IN_MICROPHONE		0x0001
-#define AUDIO_DEVICE_OUT_EARPIECE		0x0100
-#define AUDIO_DEVICE_OUT_SPEAKER		0x0200
-#define AUDIO_DEVICE_OUT_HEADSET		0x0400
+#define AUDIO_DEVICE_UNDEF 0x0000
+#define AUDIO_DEVICE_IN_MICROPHONE 0x0001
+#define AUDIO_DEVICE_OUT_EARPIECE 0x0100
+#define AUDIO_DEVICE_OUT_SPEAKER 0x0200
+#define AUDIO_DEVICE_OUT_HEADSET 0x0400
 
-#define AUDIO_DEVICE_MAX_VOLUME			10
+#define AUDIO_DEVICE_MAX_VOLUME 10
+
+#define MAX_SUPPORTED_SAMPLERATE 32
 
 /****************************************************************************
  * Private Declarations
  ****************************************************************************/
-enum audio_card_status_e {
-	AUDIO_CARD_IDLE = 0,
-	AUDIO_CARD_READY,
-	AUDIO_CARD_RUNNING,
-	AUDIO_CARD_PAUSE
-};
+enum audio_card_status_e { AUDIO_CARD_IDLE = 0, AUDIO_CARD_READY, AUDIO_CARD_RUNNING, AUDIO_CARD_PAUSE };
```

Comment:
previous is right coding style, as I know

Reply:
Yes, It was because of the formatter. Fixed.